### PR TITLE
Add skt stubs, properties

### PR DESCRIPTION
**Pull request by blluv**
- com.skt.m.ProgressBar, com.skt.m.Device::enableRestoreLCD, com.skt.m.Graphics2d::drawImage stub 추가
- m.SK_VM, com.xce.wipi.version property 추가

### CODE REVIEW

File: wie_skvm/src/classes/com/skt/m/progress_bar.rs [hunk 1]
```diff
@@ -0,0 +1,44 @@
+use alloc::vec;
+
+use java_class_proto::JavaMethodProto;
+use java_runtime::classes::java::lang::String;
+use jvm::{ClassInstanceRef, Jvm, Result as JvmResult};
+
+use wie_jvm_support::{WieJavaClassProto, WieJvmContext};
+
+// interface com.skt.m.ProgressBar
```
Comment (dlunch) [inline wie_skvm/src/classes/com/skt/m/progress_bar.rs:9]:
```suggestion
// class com.skt.m.ProgressBar
```

File: wie_skvm/src/classes/com/skt/m/progress_bar.rs [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+pub struct ProgressBar;
+
+impl ProgressBar {
+    pub fn as_proto() -> WieJavaClassProto {
+        WieJavaClassProto {
+            name: "com/skt/m/ProgressBar",
+            parent_class: None,
```
Comment (dlunch) [inline wie_skvm/src/classes/com/skt/m/progress_bar.rs:16]:
```suggestion
            parent_class: Some("java/lang/Object"),
```

File: wie_skvm/src/classes/com/skt/m/progress_bar.rs [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    }
+
+    async fn init(_jvm: &Jvm, _context: &mut WieJvmContext, this: ClassInstanceRef<Self>, name: ClassInstanceRef<String>) -> JvmResult<()> {
+        tracing::warn!("stub com.skt.m.ProgressBar::<init>({:?}, {:?})", &this, name);
+
+        Ok(())
+    }
+
+    async fn set_max_value(_jvm: &Jvm, _context: &mut WieJvmContext, this: ClassInstanceRef<Self>, value: i32) -> JvmResult<()> {
+        tracing::debug!("com.skt.m.ProgressBar::setMaxValue({:?}, {:?})", &this, value);
```
Comment (dlunch) [inline wie_skvm/src/classes/com/skt/m/progress_bar.rs:34]: Please change this and `set_value` method's log also

**Comment by blluv** [comment]
Thank you for the review. I have resolved all the issues.

**Comment by dlunch** [comment]
Could you please fix issues in ci?

**Comment by blluv** [comment]
The error in CI is due to using too many arguments in the function. would it be okay to fix this with `#[allow(clippy::too_many_arguments)]`?

**Comment by dlunch** [comment]
> The error in CI is due to using too many arguments in the function. would it be okay to fix this with `#[allow(clippy::too_many_arguments)]`?

Yes please

**Comment by blluv** [comment]
Okay, I've fixed it. Sorry for the late fix.

**Review by dlunch — APPROVED** [review]

**Pull request merged**